templates/federated_reporting/import_file.sh

```diff
@@ -57,7 +58,7 @@ EOF
 UPDATE public.__hubs SET last_import_ts = now() WHERE hostkey = '$hostkey';
 COMMIT;
 EOF
-} | "$CFE_BIN_DIR"/psql -U $CFE_FR_DB_USER -d cfdb 2>&1 | "$CFE_FR_COMPRESSOR" $CFE_FR_COMPRESSOR_ARGS >"$file.log.$CFE_FR_COMPRESSOR_EXT" && {
+} | "$CFE_BIN_DIR"/psql -v ON_ERROR_STOP=1 -U $CFE_FR_DB_USER -d cfdb 2>&1 | "$CFE_FR_COMPRESSOR" $CFE_FR_COMPRESSOR_ARGS >"$file.log.$CFE_FR_COMPRESSOR_EXT" && {
```

Comment:
Now that I'm looking at this again, I see `\set ON_ERROR_STOP 1` few lines above this changed one. Is that not working? Or shall it be moved somewhere else? It should only be set in one place.

Reply:
Ah. It is the `set -o pipefail` that makes things work correctly. Putting `ON_ERROR_STOP=1` either place will work.

Reply:
@vpodzime take another look and approve if you agree. I did some testing and figured out where the real fix was. :+1: `set -o pipefail`